CHANGELOG.md

```diff
@@ -4,6 +4,84 @@
 
 <!-- next version -->
 
+# v0.57.2
```

Comment:
Why can't we use 0.57.0 for contrib and packages/binaries even if they depend on core 0.57.2? 0.57.2 seems like a patched version for specifically for contrib. I believe we had cases where core and contrib patch versions didn't match.

Reply:
See the leads slack channel for the discussion.

Reply:
We did indeed, and we had users confused back then when they asked for a .1 version of the core binary, even though the -releases .0 incorporated core's .1 already. I think the way this PR is set generates less confusion for our end users.

Reply:
I don't think it is appropriate to skip version numbers.  I think it will be confusing to users if there are missing versions that never existed.

Reply:
It's also confusing to have versions that look the same (core 0.57.2, contrib 0.57.0, releases 0.57.0), and we had a similar situation in the past where users were asking when a binary of the core would be released, as they thought a 0.x.1 in the core would produce a 0.x.1 binary.

I'll add this as a topic to discuss during the next SIG, as there are other situations that we might want to have a consistent answer, like core 0.x.0 + contrib 0.x.1, or core/contrib 0.x.0 + releases 0.x.2.